exercises/G-numbers/exercise.js

```diff
@@ -1 +1,7 @@
-// Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 15;
+
+let numberOfMentors = 8;
```

Comment:
(this comment relates to all the uses of `let` in this PR).

If the variable value isn't going to be changed after its initial assignment, which is better, `let` or `const`?